ddp-workspace/projects/ddp-mbc/src/app/components/data-release/data-release.component.html

```diff
@@ -0,0 +1,261 @@
+                                Toolkit.DataRelease.Resources.Title
+                            </h2>
+                            <p class="PageContent-text">
+                                <span translate>Toolkit.DataRelease.Resources.TextPt1</span>
+                                <a (click)="scrollTo(glossary)" class="Link" translate>Toolkit.DataRelease.Resources.Link</a>
+                                <span translate>Toolkit.DataRelease.Resources.TextPt2</span>
+                            </p>
+                            <h2 class="PageContent-subtitle" translate>
+                                Toolkit.DataRelease.Unknown.Title
+                            </h2>
+                            <p class="PageContent-text" translate>
+                                Toolkit.DataRelease.Unknown.Text
+                            </p>
+                            <h2 class="PageContent-subtitle" translate>
+                                Toolkit.DataRelease.Samples.Title
+                            </h2>
+                            <p class="PageContent-text" translate>
+                                Toolkit.DataRelease.Samples.Text
+                            </p>
+                            <h2 class="PageContent-subtitle" translate>
+                                Toolkit.DataRelease.Sequence.Title
+                            </h2>
+                            <p class="PageContent-text" translate>
+                                Toolkit.DataRelease.Sequence.Text
+                            </p>
+                            <h2 class="PageContent-subtitle" translate>
+                                Toolkit.DataRelease.Inform.Title
+                            </h2>
+                            <p class="PageContent-text" [innerHtml]="'Toolkit.DataRelease.Inform.Text' | translate">
+                            </p>
+                            <h2 class="PageContent-subtitle" translate>
+                                Toolkit.DataRelease.Biases.Title
+                            </h2>
+                            <h2 class="PageContent-text">
+                                <span translate>Toolkit.DataRelease.Biases.Text</span>
+                                <a [href]="dataEmailHref" class="Link">{{ infoEmail }}</a>.
+                            </h2>
+                            <ul class="PageContent-ul">
+                                <li class="PageContent-text PageContent-text-list">
+                                    <a href="Methods.pdf" target="_blank" class="Link" translate>Toolkit.DataRelease.Appendices.AppendicesList.Item1.Link</a>
+                                    <span translate>Toolkit.DataRelease.Appendices.AppendicesList.Item1.Text</span>
+                                </li>
+                                <li class="PageContent-text PageContent-text-list">
+                                    <a [href]="currentLanguage === 'es' ? 'AboutYouSurveySpanish.pdf' : 'AboutYouSurveyV2.pdf'"
```

Comment:
Apply different pdf for English/Spanish languages.

No other changes in this file.